opencti-platform/opencti-graphql/src/modules/support/support-converter.ts

```diff
@@ -0,0 +1,24 @@
+import { buildStixObject, cleanObject } from '../../database/stix-converter';
+import { STIX_EXT_OCTI } from '../../types/stix-extensions';
+import type { StixSupportPackage, StoreEntitySupportPackage } from './support-types';
+
+const convertSupportPackageToStix = (instance: StoreEntitySupportPackage): StixSupportPackage => {
```

Comment:
Can this be removed ? I tried but failed... I don't think we need a Stix export.

Reply:
no we need to keep it.